utils/logger.ts

```diff
@@ -69,16 +69,30 @@ const converseTransport: transportFunctionType = async (props) => {
   await RNFS.appendFile(loggingFilePath, `${props.msg}\n`, "utf8");
 };
 
+const developerSystemAppearance: "dark" | "light" = "dark" as const;
```

Comment:
Should we put an env variable and default to "dark"?

Reply:
probably, but didn't want to spend more than 5 seconds on this lol